tests/unit/via/missing_init_files_test.py

```diff
@@ -8,7 +8,11 @@ def test_for_missing_init_files():
         if root.endswith("__pycache__"):  # pragma: no cover
             continue
 
-        if root in ["tests/pytest_plugins"]:
+        if root in [
+            "tests/pytest_plugins",
+            "tests/unit/pytest_plugins",
+            "tests/functional/pytest_plugins",
+        ]:
```

Comment:
The missing `__init__.py` files test needs to ignore the cookiecutter's pytest plugins directories